app/models/cart.rb

```diff
@@ -24,6 +24,7 @@ def items
     end
   end
 
```

Comment:
added in notes about which cart functions may be useful for adding the discount functionality to items